docs/src/components/home/sections/PrimitiveSection.tsx

```diff
@@ -0,0 +1,245 @@
+            <SwitchField label="switch" labelPosition="end" />
+          </Center>
+        </Card>
+        <Card variation="elevated" flex="1" display={['none', 'block']}>
+          <Center>
+            <Pagination
+              currentPage={currentPageIndex}
+              totalPages={totalPages}
+              siblingCount={1}
+              onChange={handleOnChange}
+            />
+          </Center>
+        </Card>
+      </Flex>
+      <Flex direction={['column', 'row']}>
+        <Card variation="elevated" flex="1">
+          <Center>
+            <Button>Button</Button>
+            <Button variation="primary">Button</Button>
+            <Button variation="link">Button</Button>
+          </Center>
+        </Card>
+        <Card variation="elevated" flex="1">
+          <Flex
+            direction="row"
+            justifyContent="center"
+            alignContent="center"
+            height="100%"
+          >
+            <ToggleButtonGroup
+              value={exclusiveValue}
+              isExclusive
+              onChange={(value: string) => setExclusiveValue(value)}
+            >
+              <ToggleButton value="align-left" ariaLabel="align left">
+                <MdFormatAlignLeft />
+              </ToggleButton>
+              <ToggleButton value="align-center" ariaLabel="align center">
+                <MdFormatAlignCenter />
+              </ToggleButton>
+              <ToggleButton value="align-right" ariaLabel="align right">
+                <MdFormatAlignRight />
+              </ToggleButton>
+              <ToggleButton value="align-justify" ariaLabel="align justify">
+                <MdFormatAlignJustify />
+              </ToggleButton>
+            </ToggleButtonGroup>
+          </Flex>
+        </Card>
+      </Flex>
+      <Flex direction={direction}>
+        <Card variation="elevated" flex="2">
+          <Tabs>
+            <TabItem title="Sports"></TabItem>
+
+            <TabItem
+              title={
+                <View>
+                  Donuts{' '}
+                  <Badge size="small" variation="success">
+                    new
+                  </Badge>
+                </View>
+              }
+            ></TabItem>
+          </Tabs>
+        </Card>
+        <Card variation="elevated" flex="1">
+          <Center>
+            <SearchField placeholder="Search" label="Search" />
+          </Center>
+        </Card>
+      </Flex>
+      <Flex direction={direction}>
+        <Card variation="elevated" flex="1">
+          <Center>
+            <RadioGroupField
+              label="Language"
+              name="language"
+              value={radioValue}
+              direction="row"
+              gap="small"
+              onChange={(e) => setRadioValue(e.target.value)}
+              labelHidden
+            >
+              <Radio value="html">html</Radio>
+              <Radio value="css">css</Radio>
+              <Radio value="javascript">javascript</Radio>
```

Comment:
nit: Can we adjust the capitalization for the displayed text? `HTML`, `CSS`, `JavaScript`